### PR TITLE
{tools}[dummy/dummy] py-cpuinfo v5.0.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3111.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/7a340f81a7b508407003a4eeac478e2a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in this PR)
boegels-MacBook-Pro.local - Darwin UNKNOWN UNKNOWN, Intel(R) Core(TM) i5-7360U CPU @ 2.30GHz, Python 2.7.10
See https://gist.github.com/d25fd4be01061dc683ad3773a4edf6ef for a full test report.

File: easybuild/easyconfigs/p/py-cpuinfo/py-cpuinfo-5.0.0.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['2cf6426f776625b21d1db8397d3297ef7acfa59018f02a8779123f3190f18500']
+
+allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+pyshortver = '.'.join(SYS_PYTHON_VERSION.split('.')[:2])
+
+# don't use pip since we're not sure whether or not it's installed on the system...
+use_pip = False
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/py-cpuinfo/py-cpuinfo-5.0.0.eb]: Travis says missing download_dep_fail

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2006.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/53fcc13ffa740235e5504cb05b8a7ce7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
nata125.ugent.be - Darwin UNKNOWN UNKNOWN, Intel(R) Core(TM) i5-7360U CPU @ 2.30GHz, Python 2.7.10
See https://gist.github.com/e9dc7f3b24f6a04655c94c4ed2516864 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2406.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/089a649c3a0fff47b9747d0186fa2c45 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3102.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/e3e8fe5e9eff2a37bf553f7c2ebc6627 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/d7dd6a84ca5bb44d74b5cf8c5c399d58 for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @boegel!

**Pull request merged**